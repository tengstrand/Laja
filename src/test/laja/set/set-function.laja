
#function value()
  #return 123
#end

#write "target/test-output/set-function.out"
  #set (x = value())
  #set (compare = value().compareTo(2))
x = {x}
compare = {compare}
#end