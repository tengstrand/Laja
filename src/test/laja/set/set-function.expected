x = 123
compare = 1
